cpanfile

```diff
@@ -5,12 +5,14 @@ use warnings;    # satisfy linter
 if ( "$]" < 5.012 ) {
     requires 'Data::Section', '==0.200007';
     requires 'Data::OptList', '==0.113';
+    requires 'Plack', '==1.0050';
```

Comment:
Plack::Test is indeed provided by Plack

Reply:
I initially had `Plack::Test` with a version requirement, but it did not resolve via `cpm`. I think because `Plack::Test` doesn't contain a `$VERSION`.